src/main/app/case/CaseApi.ts

```diff
@@ -192,7 +192,7 @@ export class CaseApi {
    * @returns
    */
   private async sendEvent(caseId: string, data: Partial<CaseData>, eventName: string): Promise<CaseWithId> {
-    console.log({ caseId, data, eventName });
+    // console.log({ caseId, data, eventName });
     return new Promise(() => {
```

Comment:
Would it not be better simply to delete the logging, instead of commenting it out?